### PR TITLE
adding custom sample size

### PR DESCRIPTION
**Pull request by MorganLove**
**Description**

- The custom sample size feature got lost in the mix, and now it is added back in again.

**Testing**

- Snapshots updated

**Progress**

- lgtm, but haven't tested it exhaustively.

### CODE REVIEW

File: client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx [hunk 1]
```diff
@@ -246,4 +246,48 @@ describe('Audit Setup > Review & Launch', () => {
       })
     })
   })
+
+  it.only('accepts custom sample size', async () => {
```
Comment (jonahkagan) [inline client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx:250]: supposed to be `.only`?

File: client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx [hunk 2]
```diff
@@ -246,4 +246,48 @@ describe('Audit Setup > Review & Launch', () => {
...
+    const { getByText, findByText, getAllByText } = render(
+      <Router>
+        <Review locked={false} prevStage={prevStage} refresh={jest.fn()} />
+      </Router>
+    )
+    const newSampleSize = await findByText(
+      'Enter your own sample size (not recommended)'
+    )
+    fireEvent.click(newSampleSize, { bubbles: true })
+    const customSampleSizeInput = await screen.findByRole('textbox')
```
Comment (jonahkagan) [inline client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx:267]: any reason you're using `screen` here but not in the rest of the test?

File: client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx [hunk 3]
```diff
@@ -246,4 +246,48 @@ describe('Audit Setup > Review & Launch', () => {
...
+    fireEvent.change(customSampleSizeInput, { target: { value: '40' } })
+    fireEvent.blur(customSampleSizeInput)
+    await screen.findByText(
+      'Must be less than or equal to the total number of ballots in targeted contests'
+    )
```
Comment (jonahkagan) [inline client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx:272]: nice test

File: client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx (hunk 2 above)
Comment (MorganLove) [reply client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx:267]: @jonahkagan I was just adding the new test in, without updating the others yet, so I used the new features for the new test. I could probably easily swap out that bit, though.

File: client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx (hunk 1 above)
Comment (MorganLove) [reply client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx:250]: @jonahkagan Ah, thanks for the catch. I was doing that to speed up test development, forgot to take it back out again, haha.

**Pull request merged**